app/controllers/collections_controller.rb

```diff
@@ -8,31 +8,23 @@ class CollectionsController < ApplicationController
   include KwicConcordanceConcern
 
   before_action :require_editor, except: %i(show download print kwic kwic_download)
-  before_action :set_collection, only: %i(show edit update destroy)
-
-  # GET /collections or /collections.json
-  def index
-    @collections = Collection.all
-  end
+  before_action :set_collection, only: %i(show update destroy)
 
   # GET /collections/1 or /collections/1.json
   def show
-    unless @collection.has_multiple_manifestations?
-      ci = @collection.flatten_items.select { |x| x.item_type == 'Manifestation' }.first
-      if ci.nil?
-        flash[:error] = t(:no_such_item)
```

Comment:
We've discussed this over Telegaram and agreed it was a bug: if there were no manifestations in collection (including subcollections) then 'no such item' message was shown.